napari/components/_layers/_base_layer/model.py

```diff
@@ -5,7 +5,7 @@
 
 import weakref
 
-from ...util.event import Event
+from ....util.event import Event
```

Comment:
Honestly, to me this says we have gone in too deep, and makes me :-1: on this PR. I don't think I've ever seen four leading dots in a Python import before.

Reply:
Agreed I don't love this either, but then I'd rather see us fix our `util.event` import (and other weird imports). To me the new organization really makes much more sense, and I think will be easier for new-comers to grasp. @kne42 what is your opinion?